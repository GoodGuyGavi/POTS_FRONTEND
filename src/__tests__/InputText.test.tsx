/**
 * @format
 */

import React from "react";
import { shallow } from "enzyme";
import InputText from "../components/InputText";

describe("Input Text Form", () => {
  test("renders correctly", () => {
    expect(
      shallow(
        <InputText type="text" title="With Label" label="This is Label:" />
      )
    ).toMatchSnapshot();
  });
});
