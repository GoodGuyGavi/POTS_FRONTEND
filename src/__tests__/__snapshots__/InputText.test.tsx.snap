// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Text Form renders correctly 1`] = `ShallowWrapper {}`;
